### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by cmartin81** [networknt/angular-schema-form-marked#1]
I can't get it to work.
Do you have a complete schema and form objects that works?

**Comment by stevehu** [comment]
Yes. It works. I just added a sample web app and gulp-webserver to demo it.

clone the repo and run the following to see how it works.

npm install
bower install
gulp webserver

Thanks.
